### ISSUE TITLE
Broken grid layout (columns missing values)

### ISSUE THREAD

**Issue by novoj** [lukashornych/evitalab#121]
[This URL](https://demo.evitadb.io/lab?sharedTab=N4IgLghgRgKgngBwKYgFwgCYUgWgOYBOAlhiADTjQAKEBEAtgM5qgDGA9gHadKthFcAkqXQYk9duRCtsEADbs8AOQYp0SAG5FIUpJ35g48ZGhABhbEjzsCcEAF8KkKABFZLEAEcArklsAZCE48bwg8NRBNbQhPOSkAMyI5MD8AITt0bDBiKG8Usy5IIk5GAAoAHWl2MUqyAAJKpFpKgEopGzECdNMpLEh-dhk5CL1eokYEOQg4JAwqAnZkAn4kZlQAbRAEYnpaOABpJDsKBFo9MHmiXdtD45AFIdWpeTkBx+YKLJy81dQCJCmKQwADEkk9PmBskRcilGKgOGJnpDvrDUFpGNCkto7l9oT84cMNACkVCYb8MONJtMAEpIeJ+PSsWYWFLWWzzarePgfEC4slwsSMVjEBD8LgklG-BDeKBycYAC1mErxqMFwqIooEnAAyvKbGBlfzUN4CHEIaT8fDLGyjIgUObJXCTXJBJwIHwiETDZa3fR7bzkSrfhp5CRsd7UYxIGBvDy+ZbWPKguFSA6g3COn4I78o9hY9m4ejMXLDCAALonMJIbVEABeagATABWSvhJTeehQLOoACM9nsQA) incorrectly displays the values in the columns.

![image](https://github.com/lukashornych/evitalab/assets/202912/5a8de7a0-3eda-49ed-8ee8-e7cd43831047)

I've selected Categories and added attributes `status` and `visibility` to the default listing.

**Comment by lukashornych** [comment]
Interesting problem. The URL is indeed broken, however, I don't know how to replicate it. There seem to duplicate columns but how did you add the `status` attribute? This one is already preselected by lab and toggling it off and on again doesn't broke it :confused:.

**Comment by novoj** [comment]
I selected them via the standard dialog:

![image](https://github.com/lukashornych/evitalab/assets/202912/59a7e3d5-23f1-444c-bb63-542a8db7cc1e)

They were not selected there ... and I managed to do it for the second time:

![image](https://github.com/lukashornych/evitalab/assets/202912/a701cae3-af20-4b3e-9c44-36d7dab3c606)

**Comment by lukashornych** [comment]
@novoj Could you check if you can still replicate it, please? I'm not able to replicate it anymore. Maybe it was fixed with other issues.

**Comment by novoj** [comment]
No, I can't on demo. The link now displays:

![image](https://github.com/lukashornych/evitalab/assets/202912/137a7533-504e-40f6-8ebe-9affffb93f05)

**Comment by lukashornych** [comment]
@novoj Yeah, the link is broken, I know. Are able to replicate it from scratch?

**Comment by novoj** [comment]
No I'm not. It's been quite a while. Selecting columns now works correctly.

**Comment by lukashornych** [comment]
Ok, thanks for checking. I will close it for now.

**Comment by lukashornych** [comment]
Fixed, the problem occured when the grid is loaded from shared tab. In this case the entity properties that should be preselected from the shared tab weren't properly initialized (mainly because of JS missing equals methods).

**Comment by novoj** [comment]
Good job!
